config/config.go

```diff
@@ -54,6 +54,23 @@ func (d *DefaultConfigDirectoryFetcher) GetUserConfigDir() (string, error) {
 	}
 }
 
+// Takes as input the user defined sessions from their config
+// and cleans all paths to be absolute
+func (c *Config) CleanPaths() error {
+	switch runtime.GOOS {
+	case "windows":
+		return nil
+
+	default:
+		for idx, sessionConfig := range c.SessionConfigs {
+			// convert user home dirs to absolute paths
+			c.SessionConfigs[idx].Path = dir.AlternatePath(sessionConfig.Path)
+			c.SessionConfigs[idx].StartupScript = dir.AlternatePath(sessionConfig.StartupScript)
+		}
```

Comment:
I noticed you're converting the sessionConfig paths to absolute paths lazily (at the moment its needed) in a couple of places.
For my Path+Name comparison of config sessions and active tmux sessions I needed to convert to absolute again.

I figured that this might rather be done only once greedily (at parse time).